.github/workflows/e2e-certifier.yml

```diff
@@ -0,0 +1,34 @@
+name: 'End-to-end certifier'
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment in which to run the job
+      region:
+        description: The region in which to run the job
+      package:
+        description: The package name
+      job:
+        description: The name of the job (as defined in the deployment config)
```

Comment:
These values are not used in the jobs, but they're required by the deployment pipeline: https://coveord.atlassian.net/wiki/spaces/CM/pages/3980165140/Team+GitHub+Phase